fetch.bs

```diff
@@ -1292,28 +1292,45 @@ run these steps:
 
 <div algorithm>
 <p>To <dfn id=simple-range-header-value>parse a single range header value</dfn> from a
-<a>byte sequence</a> <var>value</var>, run these steps:
+<a>byte sequence</a> <var>value</var> and a boolean <var>disallowWhitespace</var>, run these steps:
```

Comment:
Since it's a required argument, perhaps we should invert this to not have a double negative.

Reply:
👍 yeah that makes sense